dls-controls/malcolmjs
# Feature/document code structure #193

## Description

Added a mechanism to automatically acquire screenshots for the SMG and the User Guide.
Added a description of the code structure.
Added a quick start to the user guide.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Review the built documentation

## Agile board tracking

connect to #193 
connect to #197 
closes #193 
closes #197
